Backpack/Chip/Classes/BPKChip.m

```diff
@@ -155,7 +168,7 @@ - (void)setUpConstraints {
 
 - (void)updateStyle {
     if (self.selected) {
-        self.backgroundColor = BPKColor.blue500;
+        self.backgroundColor = self.primaryColor;
```

Comment:
Since it's `nullable` maybe we should also have a default here?

Reply:
I'll just make them non-nullable as they shouldn't be anyway 👍